wrappers/python/examples/box_dimensioner_multicam/realsense_device_manager.py

```diff
@@ -40,9 +40,13 @@ def enumerate_connected_devices(context):
 
     """
     connect_device = []
+
     for d in context.devices:
         if d.get_info(rs.camera_info.name).lower() != 'platform camera':
-            connect_device.append(d.get_info(rs.camera_info.serial_number))
+            serial = d.get_info(rs.camera_info.serial_number)
+            product_line = d.get_info(rs.camera_info.product_line)
+            device_info = (serial, product_line) # (serial_number, product_line)
+            connect_device.append( device_info )
     return connect_device
```

Comment:
Update Return in the doc -- it's now an array of (serial, product-line) tuples

Reply:
done